osu.Game.Rulesets.Taiko/Mods/TaikoModHealthChallenge.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Rulesets.Mods;
+
+namespace osu.Game.Rulesets.Taiko.Mods
+{
+    public class TaikoModHealthChallenge : ModHealthChallenge
+    {
+
+    }
```

Comment:
If all ruleset-specific implementations of an abstract mod are empty, then I'd say that the implementations are useless and the base abstract mod should just become non-abstract and be used instead. And yes I know that `ModPerfect` does what this does, and I also think that's a mistake.